ubuntudesign/developer.ubuntu.com
# Reposition search

## Done

Reposition search and make header taller

## QA

- Run ./run to get started
- Go to http://localhost:8015/core
- Make sure search is right aligned on large and that small works as expected
- The styling is as per live but I’m going to take a second pass at this to make it better
